Dynavity/Dynavity/view/CanvasElementView.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  CanvasElementView.swift
+//  Dynavity
+//
+//  Created by Hans Sebastian Tirtaputra on 15/3/21.
+//
+
+import SwiftUI
+
+struct CanvasElementView: View {
+    static let width = CGFloat(150) // arbitrary value for now
```

Comment:
Could we mark this as a `TODO` so that it is obvious that this should be changed?

Reply:
Got it, thanks!